### PR TITLE
Fix Xcode 8.3.3 Empty paragraph passed to '@param' command warnings

### PR DESCRIPTION
**Pull request by musbaalbaki**

### CODE REVIEW

File: Classes/UIScrollView+InfiniteScroll.m [hunk 1]
```diff
@@ -34,7 +34,7 @@ static void PBSwizzleMethod(Class c, SEL original, SEL alternate) {
  *
  *  See https://github.com/pronebird/UIScrollView-InfiniteScroll/issues/31
  *
- *  @param tableView
+ *  @param tableView A helper function to force table view to update its content size.
```
Comment (pronebird) [inline Classes/UIScrollView+InfiniteScroll.m]: Hi @musbaalbaki,

Thanks for your PR. But the parameter description does not reflect the parameter itself. In that case it's tableView which is not a function.

Comment (musbaalbaki) [inline Classes/UIScrollView+InfiniteScroll.m]: My bad @pronebird, that's correct. Will fix that. Thanks

Comment (pronebird) [inline Classes/UIScrollView+InfiniteScroll.m]: I will fix it shortly

**Comment by pronebird** [comment]
Closed via #63

**Pull request closed without merging**